src/main/scala/uk/gov/hmrc/testreport/TestReportPlugin.scala

```diff
@@ -40,6 +40,9 @@ object TestReportPlugin extends AutoPlugin {
   )
 
   private def generateTestReport(): Def.Initialize[Task[Unit]] = Def.task {
+    val log = sbt.Keys.streams.value.log
+    log.info("Generating accessibility assessment report")
```

Comment:
Really trivial one but should we add `...` to the output similar to other log lines where items are loading? e.g. `Generating accessibility assessment report...`